stubs/openpyxl/openpyxl/xml/_functions_overloads.pyi

```diff
@@ -16,8 +16,10 @@ _T_co = TypeVar("_T_co", covariant=True)
 
 # Comment from openpyxl.cell.rich_text.py
 # Usually an Element() from either lxml or xml.etree (has a 'tag' element)
+# lxml.etree._Element
+# xml.etree.Element
 class _HasTag(Protocol):
-    tag: Any  # AnyOf[str, None, Callable[..., AnyOf[str, None]]]
```

Comment:
This strange Union was caused by `Element`'s typing leading me to believe it could be `Callable` and the runtime implementation leading me to believe it could be `None` (see PR description).
Looking at xml.etree._Element typing from lxml-stubs as well as implementation of openpyxl looks like this is never assumed to be `None` neither.